.travis.yml

```diff
@@ -1,30 +1,43 @@
-sudo: false
-cache: pip
+language: python
+cache:
+  directories:
+  - $HOME/.cache/pip
```

Comment:
Is this change needed for the new stuff?

Reply:
caching `$HOME/.cache/pre-commit` helps performance of the linter job.